docs/content/documentation/systems/BCs/xfem/CrackTipEnrichmentCutOffBC.md

```diff
@@ -0,0 +1,12 @@
+
+# CrackTipEnrichmentCutOffBC
+MOOSE currently does not permit to add variables on a subset of the domain. An in-efficient workaround is to add new variables associated with the enrichment shape functions for the whole mesh and then to simply fix DOFs to zero for those nodes with basis function supports that are far away from any crack tip. This approach is implemented in MOOSE for simplicity and parameter `cut_off_radius` is used to specify the cut-off radius for the enrichment variables.
```

Comment:
I'd reword the first couple sentences like this:

MOOSE permits variables to be restricted to specific blocks, but currently does not provide a straightforward way to add variables to just the enriched region near the crack tip. An inefficient workaround is to add new variables associated with the enrichment shape functions for the whole mesh and then to simply fix DOFs to zero for those nodes with basis function supports that are far away from any crack tip.

Reply:
Done